internal/utils/meta.go

```diff
@@ -31,3 +33,14 @@ func AddAnnotation(obj metav1.Object, key string, value string) bool {
 	}
 	return false
 }
+
+// HasOwnerReference returns true if the object has an owner ref
```

Comment:
`HasOwnerReference` to `IsOwnedBy`

Reply:
Good catch
I started with the `HasOwnerReference` name but it is too generic as it means "has any owner ref", then I changed it to `IsOwnedBy`. It seems I forgot to update the comment